tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive_IPI.py

```diff
@@ -126,47 +139,56 @@ def test_automated_recovery_from_failed_nodes_IPI_reactive(
         if failure == "shutdown":
             nodes.stop_nodes(failure_node_obj, wait=True)
             log.info(
-                f"Successfully powered off node: {failure_node_obj[0].name}"
-            )
-            nodes.terminate_nodes(failure_node_obj, wait=True)
-            log.info(
-                f"Successfully terminated node : {failure_node_obj[0].name} instance"
+                f"Successfully powered off node: "
+                f"{failure_node_obj[0].name}"
             )
         elif failure == "terminate":
             nodes.terminate_nodes(failure_node_obj, wait=True)
             log.info(
-                f"Successfully terminated node : {failure_node_obj[0].name} instance"
+                f"Successfully terminated node : "
+                f"{failure_node_obj[0].name} instance"
             )
 
-        # DC app pods on the failed node will get automatically created on other
-        # running node. Waiting for all dc app pod to reach running state
-        pod.wait_for_dc_app_pods_to_reach_running_state(dc_pod_obj)
-        log.info("All the dc pods reached running state")
+        try:
```

Comment:
Hi @PrasadDesala , please review if this change is ok with this test case you created